lexnlp/extract/es/tests/test_regulations.py

```
@@ -67,3 +70,66 @@ def test_file_samples(self):
             get_regulation_annotations,
             'lexnlp/typed_annotations/es/regulation/regulations.txt',
             RegulationAnnotation)
+
+
+# ---------------------------------------------------------------------------
+# RegulationsParser – DataFrame injection (PR fix: `is None` check)
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)